src/cascadia/TerminalApp/AppLogic.h

```diff
@@ -127,7 +129,13 @@ namespace winrt::TerminalApp::implementation
 
         Windows::Foundation::Collections::IMapView<Microsoft::Terminal::Control::KeyChord, Microsoft::Terminal::Settings::Model::Command> GlobalHotkeys();
 
+        Microsoft::Terminal::Settings::Model::Theme Theme();
+
         // -------------------------------- WinRT Events ---------------------------------
+        // PropertyChanged is surprisingly not a typed event, so we'll define that one manually.

```

Comment:
W A T

Reply:
oh butts it should be `WINRT_CALLBACK(PropertyChanged, Windows::UI::Xaml::Data::PropertyChangedEventHandler);
`